aif360/metrics/utils.py

```diff
@@ -1,12 +1,6 @@
-"""This is the helper script for implementing metrics."""
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-from __future__ import unicode_literals
-
+import sklearn
 import numpy as np
 
-
```

Comment:
you can leave 2 lines after the imports for consistency with the rest of the files